Sources/App/CRP/CRPProcess.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+import Vapor
+import Stevenson
+
+enum CRPProcess {
```

Comment:
offtopic: now I'm more convinced we should have this in another app that would use Stevenson framework for Slack integration

Reply:
Yeah agreed, we might want to redefine the responsibilities for Stevenson bot 👍 
Maybe once we first merge our implementations of GitHub/CircleCI/… APIs one day between Stevenson and Wall-E and end up with a stronger reusable framework?